nixos/modules/services/x11/window-managers/oroborus.nix

```diff
@@ -0,0 +1,28 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.xserver.windowManager.oroborus;
+in
+{
+  ###### interface
+  options = {
+    services.xserver.windowManager.oroborus.enable = mkEnableOption {
```

Comment:
The syntax of `mkEnableOption` is `mkEnableOption "service-name"`, it does not take an attrset, it returns one.